### PR TITLE
Issue #3627: Add a new, optional footer variable in theme_table() that produces tfoot

### PR DESCRIPTION
**Pull request by klonos**
https://github.com/backdrop/backdrop-issues/issues/3627

Credits for this should go to @drupol

### CODE REVIEW

**Comment by drupol** [comment]
Thanks !!!

**Comment by klonos** [comment]
The PR sandbox failed to build:

Screen Shot 2019-03-28 at 8 32 00 pm

Refreshing...

File: core/includes/theme.inc [hunk 1]
```diff
@@ -2186,57 +2191,66 @@ function theme_table($variables) {
...
+  // Rows and footer have the same structure.
+  $sections = array(
+    'rows' => 'tbody',
+    'footer' => 'tfoot',
+  );
+  $variables['rows'] = $rows;
+  foreach ($sections as $section => $tag) {
+    if (!empty($variables[$section])) {
+      // <tbody> or <tfoot>.
+      $output .= "<" . $tag . ">\n";
```
Comment (drupol) [inline core/includes/theme.inc:2203]: We should use single quote.

**Review by drupol — CHANGES_REQUESTED** [review]

File: core/includes/theme.inc (hunk 1 above)
Comment (drupol) [inline core/includes/theme.inc:2203]:
```suggestion
      $output .= '<' . $tag . ">\n";
```

Comment (drupol) [reply core/includes/theme.inc:2203]: I'm not sure about keeping those `\n` in there... what do you think ?

**Comment by indigoxela** [comment]
This has been fixed in https://github.com/backdrop/backdrop-issues/issues/5042, closing as outdated.

**Pull request closed without merging**